src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        final Bulldozer bulldozer1 = new Bulldozer();
```

Comment:
```suggestion
        Bulldozer firstBulldozer = new Bulldozer();
```
change name like this